recipe/meta.yaml

```diff
@@ -283,6 +283,7 @@ outputs:
         - pyviz_comms
         - pyvo
         - ripgrep
+        - ruff
```

Comment:
We need this in rubin-env core don't we? Otherwise sconsUtils can't run the linker.

Reply:
Oops, was in the wrong section editing at the time.  Fixed now.